bitwarden_license/src/Commercial.Core/SecretManagerFeatures/AccessTokens/CreateAccessTokenCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.SecretManagerFeatures.AccessTokens.Interfaces;
+using Bit.Core.Utilities;
+
+namespace Bit.Commercial.Core.SecretManagerFeatures.AccessTokens;
+
+public class CreateAccessTokenCommand : ICreateAccessTokenCommand
+{
+    private readonly int _clientSecretMaxLength = 30;
```

Comment:
It's technically not the max length just the length 😁

Reply:
Resolved